### PR TITLE
Better markup in topic configuration help popup

### PR DESCRIPTION
**Pull request by jvorcak**
Screenshot 2024-11-06 at 10 42 19

### CODE REVIEW

**Comment by jvorcak** [comment]
Fixes https://github.com/redpanda-data/console/issues/1087

**Comment by weeco** [comment]
@jvorcak I think we can remove the "Value" row because it's already shown in the table where we show the tooltip. And maybe we can incorporate the source as a sentence after the description (coming from the backend) to get closer to the desired mock.

So just append a string on the description like this (pseudo code). I think this allows us to get rid of the value and source in this tooltip right?

```
switch topicConfig.Source {
  case DEFAULT_CONFIG:
    return "Inherited from DEFAULT_CONFIG".
  case DYNAMIC_TOPIC_CONFIG:
    return "This is a custom setting for this topic"
  case DYNAMIC_BROKER_CONFIG, STATIC_BROKER_CONFIG:
    return "This is a custom setting set on the BROKER_CONFIG level.
  default:
    return ""
}
```

**Comment by jvorcak** [comment]
Screenshot 2024-11-06 at 12 51 08

@weeco wdyt?

File: frontend/src/components/pages/topics/TopicConfiguration.tsx [hunk 1]
```diff
@@ -382,23 +361,37 @@ export const ConfigEntryEditor = observer((p: {
...
         case 'STATIC_BROKER_CONFIG':
-            return 'Set on the broker by either a config file or environment variable';
-
+            return 'This is a custom setting set on the BROKER_CONFIG level.';
         default:
             return '';
     }
 }
+
+// function getConfigSourceExplanation(source: string) {
```
Comment (weeco) [inline frontend/src/components/pages/topics/TopicConfiguration.tsx:378]: @jvorcak Do you want to remove this prior to merging?

**Review by weeco — APPROVED** [review]

**Pull request merged**